src/screens/SendTokens.js

```diff
@@ -244,11 +249,12 @@ class SendTokens extends React.Component {
     if (missingSigs.length !== 0) {
       // there are tokens without signatures, missingSigs
       // set tittle and content
-      this.setState({
-        ledgerAlertModalTitle: t`Unverified custom tokens`,
-        ledgerAlertModalBody: this.renderAlertTokenList(missingSigs),
+      this.context.showModal(MODAL_TYPES.ALERT, {
+        id: 'ledgerAlertModal',
+        title: t`Unverified custom tokens`,
+        body: this.renderAlertTokenList(missingSigs),
+        buttonName: 'Ok',
```

Comment:
The `buttonName` before was 'Cancel', or not?

Reply:
We're both wrong, the buttonName should be `Closed`

Fixed in 4155a44519e9e2a4de667e7564f976f7753f02af